src/services/legal-services.ts

```diff
@@ -291,24 +308,135 @@ export default class LegalServices {
    * @returns a promise to return the business info
    */
   static async fetchBusinessInfo (businessId: string): Promise<BusinessIF> {
-    const url = `businesses/${businessId}`
+    const url = `${this.legalBusinessUrl}businesses/${businessId}`
 
-    return axios.get(url).then(response => {
-      const data = response?.data
-      if (!data) throw new Error('Invalid API response')
-      return data.business
-    })
+    return axios.get(url, this.config)
+      .then(response => {
+        const data = response?.data
+        if (!data) throw new Error('Invalid API response')
+        return data.business
+      })
   }
 
   /**
    * Fetches the current account's authorized actions (permissions).
    * @returns a promise to return the list of authorized actions
    */
   static async fetchAuthorizedActions (): Promise<AuthorizedActions[]> {
-    return axios.get('permissions').then(response => {
-      const data = response?.data
-      if (!data) throw new Error('Invalid API response')
-      return data.authorizedPermissions
-    })
+    const url = `${this.legalBusinessUrl}permissions`
+
+    return axios.get(url, this.config)
+      .then(response => {
+        const data = response?.data
+        if (!data) throw new Error('Invalid API response')
+        return data.authorizedPermissions
+      })
+  }
+
+  /**
```

Comment:
The functions below were originally in Document Mixin but I moved them here to keep Legal API functions together.